test/unittest/tests/test_reconciliation_tracking.cpp

```diff
@@ -13,24 +13,6 @@
 #include "wrappers/connection_wrapper.h"
 #include "wt_internal.h"
 
-TEST_CASE("Reconciliation tracking: ovfl_track_init", "[reconciliation]")
-{
-    ConnectionWrapper conn(DB_HOME);
-    WT_SESSION_IMPL *session = conn.createSession();
-
-    WT_PAGE p;
-    memset(&p, 0, sizeof(p));
-
-    WT_PAGE_MODIFY m;
-    memset(&m, 0, sizeof(m));
-
-    p.modify = &m;
-
-    REQUIRE(__wt_ovfl_track_init(session, &p) == 0);
```

Comment:
If you make an "\_\_ut_" version of this function as I suggested in an earlier comment, we can keep this test and also the test below.

Reply:
Done